### PR TITLE
[READY]RychenkovAD_lab1

### PR DESCRIPTION
**Pull request by MercyFlesh**

### CODE REVIEW

File: RychenkovAD/lab1/Проект1/Source.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	char sign;
+	setlocale(0, "rus");
+	cout << "������ ������ �����: ";
+	cin >> first;
+	cout << "������ ������ �����: ";
+	cin >> second;
+	cout << "������ ��������: ";
+	cin >> sign;
+	
+	    switch (sign) {
```
Comment (Toxa-man) [inline RychenkovAD/lab1/Проект1/Source.cpp]: Bad code formatting

File: RychenkovAD/lab1/Проект1/Source.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			if (second == 0) {
+				flag = false;
+				cout << "������ �����!" << endl; break;
+			}
+			else {
+				resault = first / second; break;
+			}
+		default:
+		   {
+	       flag = false;
```
Comment (Toxa-man) [inline RychenkovAD/lab1/Проект1/Source.cpp]: Bad code formatting

File: RychenkovAD/lab1/Проект1/Source.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		   cout << "������ �����!" << endl;
+		}
+	}
+	if (flag) {
+	cout << first << " " << sign << " " << second << " " << "=" << " " << resault << endl;
```
Comment (Toxa-man) [inline RychenkovAD/lab1/Проект1/Source.cpp]: Bad code formatting. Use tabulation to highlight if-statement body

**Comment by MercyFlesh** [comment]
fixed the code format

**Pull request merged**